components/homePage/BlogCard.tsx

```diff
@@ -6,13 +6,11 @@ import {
   CardActions,
   CardContent,
   Typography,
-  Modal,
-  DialogContent,
+ 
 } from "@mui/material";
-import Image from "next/image";
 import * as React from "react";
 import { montserrat } from "../../app/fonts";
-import erroimg from "../../public/images/homePage/dialogBox/thumbDown.webp";
+import MessageBox from "./MessageBox";
```

Comment:
You have imported MessgeBox but it is not used in the file. Please remove the import if you are not using it.